### ISSUE TITLE
Add ability to read half-packets for tiebreakers

### ISSUE THREAD

**Issue by ryanrosenberg** [alopezlago/MODAQ#265]
It would be nice for a mod to have an ability to only read the first or second half of the uploaded packet. Possibly this could be accomplished by adding a Start Tossup # and End Tossup # input in the Start Game dialog once the packet has been uploaded (perhaps hidden behind a Read Half-Packet checkbox)?

**Comment by alopezlago** [comment]
I think this would also be doable as a game format by setting `regulationTossupCount` to 10. That would unblock other apps using this as a library, although the demo app doesn't have an easy way to import game formats yet (until #216 is made, perhaps).

**Comment by alopezlago** [comment]
Marking as low priority since there is a way to enforce this right now (change regulationTossupCount)

**Comment by alopezlago** [comment]
Going to Won't Fix this, I think it's too low priority to ever truly get tackled, and there is a workaround.
